spec/models/suspended_email_spec.rb

```diff
@@ -37,7 +37,8 @@
           email_address: blocked_email,
         )
 
-        expect(SuspendedEmail.find_with_email('blocked@example.com')).to eq(blocked_email)
+        email_address = SuspendedEmail.find_with_email('blocked@example.com')&.email_address
+        expect(email_address).to eq(blocked_email)
```

Comment:
(it's outside of what I can propose as a change)
I'd update the spec to say `it 'returns the suspended email record'` since that what it does now

so more like
```ruby
      it 'returns the SuspendedEmail' do
        blocked_email = FactoryBot.create(:email_address, email: 'blocked@example.com')
        digested_base_email = SuspendedEmail.generate_email_digest('blocked@example.com')
        suspended_email = FactoryBot.create(
          :suspended_email,
          digested_base_email: digested_base_email,
          email_address: blocked_email,
        )

        expect(SuspendedEmail.find_with_email('blocked@example.com')).to eq(suspended_email)
      end
```